src/Sulu/Bundle/CategoryBundle/Category/CategoryManager.php

```diff
@@ -355,7 +355,7 @@ public function save($data, $userId, $locale, $patch = false)
         }
         if (!$patch || $this->getProperty($data, 'parent')) {
             if ($this->getProperty($data, 'parent')) {
-                $parentEntity = $this->findById($this->getProperty($data, 'parent'), $locale);
+                $parentEntity = $this->findById($this->getProperty($data, 'parent'));
```

Comment:
i removed the unnecessary locale-parameter here because it is not part of the method signature